base/commandLine.cpp

```diff
@@ -69,190 +70,190 @@ static const char USAGE_STRING[] =
 static const char HELP_STRING1[] =
 	"ScummVM - Graphical Adventure Game Interpreter\n"
 	"Usage: %s [OPTIONS]... [GAME]\n"
-	"  -v, --version            Display ScummVM version information and exit\n"
-	"  -h, --help               Display a brief help text and exit\n"
-	"  -z, --list-games         Display list of supported games and exit\n"
-	"  --list-all-games         Display list of all detected games and exit\n"
-	"  -t, --list-targets       Display list of configured targets and exit\n"
-	"  --list-engines           Display list of supported engines and exit\n"
-	"  --list-all-engines       Display list of all detection engines and exit\n"
-	"  --stats                  Display statistics about engines and games and exit\n"
-	"  --list-debugflags=engine Display list of engine specified debugflags\n"
-	"                           if engine=global or engine is not specified, then it will list global debugflags\n"
-	"  --list-all-debugflags    Display list of all engine specified debugflags\n"
-	"  --list-saves             Display a list of saved games for the target specified\n"
-	"                           with --game=TARGET, or all targets if none is specified\n"
-	"  -a, --add                Add all games from current or specified directory.\n"
-	"                           If --game=ID is passed only the game with id ID is added. See also --detect\n"
-	"                           Use --path=PATH to specify a directory.\n"
-	"  --detect                 Display a list of games with their ID from current or\n"
-	"                           specified directory without adding it to the config.\n"
-	"                           Use --path=PATH to specify a directory.\n"
-	"  --game=ID                In combination with --add or --detect only adds or attempts to\n"
-	"                           detect the game with id ID.\n"
-	"  --engine=ID              In combination with --list-games, --list-all-games, --list-targets, or --stats only\n"
-	"                           lists games for this engine. Multiple engines can be listed separated by a coma.\n"
-	"  --auto-detect            Display a list of games from current or specified directory\n"
-	"                           and start the first one. Use --path=PATH to specify a directory.\n"
-	"  --recursive              In combination with --add or --detect recurse down all subdirectories\n"
-	"  --no-exit                In combination with commands that exit after running, like --add or --list-engines,\n"
-	"                           open the launcher instead of exiting\n"
+	"  -v, --version            	Display ScummVM version information and exit\n"
```

Comment:
Please revert this while the indentation change and make the new line just protrude further, e.g. like `--output-channels=CHANNELS`